app/data_models/progress_store.py

```diff
@@ -114,7 +112,7 @@ def update_section_status(
         self, section_status: str, section_id: str, list_item_id: Optional[str] = None
     ) -> bool:
         updated = False
-        section_key = (section_id, list_item_id)
+        section_key: SectionKey = SectionKey(section_id, list_item_id)
```

Comment:
Now that this is initialised with the named tuple, you won't need to also type hint it

Reply:
Updated 👍